apps/vmq_passwd/c_src/vmq_passwd.c

```diff
@@ -401,7 +404,13 @@ int main(int argc, char *argv[])
 			create_new = true;
 		}else if(!strcmp(argv[1], "-D")){
 			delete_user = true;
-		}
+		}else if(!strcmp(argv[1], "-cf")){
+                        create_new = true;
+                        overwrite_existing = true;
+                }else{
+                  print_usage();
+                  return 1;
+                }
```

Comment:
The formatting seems off here

```suggestion
        } else if (!strcmp(argv[1], "-cf")) {
            create_new = true;
            overwrite_existing = true;
        } else {
            print_usage();
            return 1;
        }
```

Reply:
Done. Nice catch. I am not used to this tab style formatting...